test/api/core/pluginsManager/init.test.js

```diff
@@ -79,6 +79,95 @@ describe('PluginsManager', () => {
       should(pluginsManager.plugins).be.empty();
     });
 
+    it('should warn if manifest.json does not define customs errors', () => {
```

Comment:
I just don't understand this warning... (see above)

Reply:
removed it